src/wp-includes/class-wp-term-query.php

```diff
@@ -593,7 +593,7 @@ public function get_terms() {
 			);
 		}
 
-		if ( '' === $args['object_ids'] ) {
+		if ( empty( $args['object_ids'] ) ) {
```

Comment:
This makes sense as it is consistent with:

https://github.com/WordPress/wordpress-develop/blob/9d7752ecb8e1f5641b2360052f01dd29617e1e38/src/wp-includes/class-wp-term-query.php#L444

Reply:
See https://core.trac.wordpress.org/ticket/63256#comment:7

```suggestion
		if ( empty( $args['object_ids'] ) && ! is_numeric( $args['object_ids'] ) ) {
```